packages/vscode-extension/src/project/devtools.ts

```diff
@@ -219,16 +233,20 @@ export class CDPDevtoolsServer extends DevtoolsServer implements Disposable {
   private maybeInitializeConnection() {
     if (this.connection) {
       // NOTE: a single `DebugSession` only supports a single devtools connection at a time
-      return;
+      return Promise.resolve();
```

Comment:
```suggestion
      return Promise.reject(/* ... */);
```
to avoid logging the `"[Devtools] Devtools connection initialized..."` message in this case.